ballista/rust/core/src/serde/logical_plan/mod.rs

```diff
@@ -988,4 +988,17 @@ mod roundtrip_tests {
 
         Ok(())
     }
+
+    #[test]
+    fn roundtrip_approx_quantile() -> Result<()> {
```

Comment:
I think it's better to move this test to `datafusion/physical_plan/.....` path.

Reply:
This test covers code exclusively within ballista and the `roundtrip_test!` macro used here is [defined in the ballista crate](https://github.com/apache/arrow-datafusion/blob/5668be78a9ccbf5469e9e95ad070920ca5d105ba/ballista/rust/core/src/serde/logical_plan/mod.rs#L44-L61) too so I think it has to live in the ballista crate. 

This file contains many other roundtrip tests, but happy to move if it if needed 👍